lib/Conversion/PipelineToCalyx/PipelineToCalyx.cpp

```diff
@@ -394,7 +394,7 @@ class BuildOpGroups : public calyx::FuncOpPartialLoweringPattern {
 
 LogicalResult BuildOpGroups::buildOp(PatternRewriter &rewriter,
                                      memref::LoadOp loadOp) const {
-  Value memref = loadOp.memref();
+  Value memref = loadOp.getMemref();
```

Comment:
^ looks like they changed MemRef dialect to use prefixed accessors too (confirmed), change LGTM of course just noting as not part of the other two bigger changes :+1: .

Reply:
Yep thanks, should have mentioned that one as well, will add it to the commit message.